presets/4.3/rc_link/TBS_race.txt

```diff
@@ -1,13 +1,13 @@
-#$ TITLE: TBS CRSF racing
+#$ TITLE: TBS Crossfire/Tracer racing
 #$ FIRMWARE_VERSION: 4.3
 #$ CATEGORY: RC_LINK
 #$ STATUS: OFFICIAL
-#$ KEYWORDS: defaults, rc link, tracer, TBS, cfv2, 250hz, crossfire, racing, 150hz
+#$ KEYWORDS: rc link, tracer, TBS, cfv2, 250Hz, crossfire, racing, 150Hz, CRSF
 #$ AUTHOR: sugarK
-#$ DESCRIPTION: setup fo the various TBS radio links via CRSF protocol for racing
-#$ DESCRIPTION: WARNING: make ABSOLUTELY SURE that the Hardware > ADC Filter is un-checked! and both radio and the TX unit are all up to date with the latest EdgeTX, OpenTX or FreedomTX
-#$ DESCRIPTION: There are 2 current types of Crossfire protocol, CFV3 if your using a TBS radio and CFV2 if your radio runs Open/EdgeTX
-#$ DESCRIPTION: Dynamic Crossfire or locked is not recommended for racing, locking at 150hz is preferred
+#$ DESCRIPTION: Setup fo the various TBS radio links via CRSF protocol for racing.
+#$ DESCRIPTION: WARNING: make ABSOLUTELY SURE that the Hardware > ADC Filter is un-checked! and both radio and the TX unit are all up to date with the latest EdgeTX, OpenTX or FreedomTX.
+#$ DESCRIPTION: There are 2 current types of Crossfire protocol, CFV3 if your using a TBS radio and CFV2 if your radio runs Open/EdgeTX.
+#$ DESCRIPTION: Dynamic Crossfire or locked is not recommended for racing, locking at 150Hz is preferred.
```

Comment:
This line makes no sense.
It says 'or locked is not recommended' then says 'locking at 150hz is preferred.
Maybe they meant to say, "Dynamic or un-locked Crossfire modes are not recommended..."

Reply:
Yeah that's not my line, I just fixed typos :)
What exactly it should say?

Reply:
`#$ DESCRIPTION: Dynamic Crossfire or locked 50Hz is not recommended for racing, locking at 150Hz is preferred. `